scripts/biomedical/human1 data/scripts/format_genes_geneRoles.py

```diff
@@ -0,0 +1,34 @@
+def main():
+    import sys
+    import os
+    import numpy as np
+    import pandas as pd
+    
+    gene_tsv, genRoles_tsv  = sys.argv[1], sys.argv[2]
+
+    ### Format genes.tsv

```

Comment:
Its useful to have some comments for what you're doing.

Reply:
All scripts commented